services/IBF-pipeline/pipeline/lib/notifications/notify.py

```diff
@@ -25,6 +28,8 @@ def notify(countryCodeISO3):
                 msg['Subject'] = formattedInfo['subject']
                 part = MIMEText(formattedInfo['html'], "html")
                 msg.attach(part)
+                print("###### formattedInfo['html'] ######")
```

Comment:
Please remove these print statements after using them for debugging. These are not useful in production.